05/DB.test.js

```diff
@@ -1 +1,23 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('.insert()', async () => {
+
+    it('take 1 item', async () => {
+        const dataBase =new DB();
+        dataBase.insert({id: 1, name: 'devmentor'});
+        const rows = await dataBase.getRows();
+
+        expect(rows.length).toBe(1);
+    });
+
+    it('show error when id is not a number', async () => {
+        const dataBase =new DB();
+        try {
```

Comment:
Jeśli używamy `try ... catch` to powinniśmy określić ilość porównań przez `expect.assertions(1)`